app/src/main/java/com/kickstarter/ui/activities/CreatorDashboardActivity.java

```diff
@@ -41,31 +41,41 @@ protected void onCreate(final @Nullable Bundle savedInstanceState) {
     ButterKnife.bind(this);
 
     // Set up the bottom sheet recycler view.
-    this.bottomSheetAdapter = new CreatorDashboardBottomSheetAdapter();
+    this.bottomSheetAdapter = new CreatorDashboardBottomSheetAdapter(this.viewModel.inputs);
     this.bottomSheetRecyclerView.setAdapter(this.bottomSheetAdapter);
     this.bottomSheetRecyclerView.setLayoutManager(new LinearLayoutManager(this)); // todo: reuse LayoutManager?
     this.bottomSheetBehavior = BottomSheetBehavior.from(this.bottomSheetRecyclerView);
 
     this.viewModel.outputs.projectAndStats()
       .compose(bindToLifecycle())
       .compose(observeForUI())
-      .subscribe(this::createProjectDashboardFragments);
+      .subscribe(this::createProjectDashboardFragment);
 
     this.viewModel.outputs.projectsForBottomSheet()
       .compose(bindToLifecycle())
       .compose(observeForUI())
       .subscribe(this::setProjectsForDropdown);
+
+    this.viewModel.outputs.projectSwitcherProjectClickOutput()
+       .compose(bindToLifecycle())
+      .compose(observeForUI())
+      .subscribe(this::newProjectClicked);
   }
 
   private void setProjectsForDropdown(final @NonNull List<Project> projects) {
     this.bottomSheetAdapter.takeProjects(projects);
   }
 
-  private void createProjectDashboardFragments(final @NonNull Pair<Project, ProjectStatsEnvelope> projectAndStats) {
+  private void newProjectClicked(final @NonNull Project project) {
```

Comment:
mayhaps `projectSwitcherProjectClicked` would be a lil more specific (like your output name!)